controller/status.go

```diff
@@ -277,6 +263,11 @@ var (
 	}
 )
 
+// Aggregator provides methods for aggregating status.
+type Aggregator struct {
```

Comment:
https://github.com/giantswarm/inago/pull/133/files#diff-9155f40475e332d8aee80a437105af6bR279 AggregateStatus is public